lms/djangoapps/certificates/views/webview.py

```diff
@@ -44,7 +43,7 @@
     CertificateStatuses,
     CertificateHtmlViewConfiguration,
     CertificateSocialNetworks)
-
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```

Comment:
this should be moved to L25 (alphabetical order), unless it causes import issues.